### PR TITLE
added custom vision deployment template

### PR DESCRIPTION
**Pull request by avkewalr-msft**
added custom vision deployment template

### CODE REVIEW

File: src/edge/deployment.customvision.template.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          }
+        },
+        "modules": {
+          "lvaEdge": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "settings": {
+              "image": "avicvreg.azurecr.io/1.1:latest",
```
Comment (naiteeks) [inline src/edge/deployment.customvision.template.json:66]: needs to be updated to the mcr image

File: src/edge/deployment.customvision.template.json [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                }
+              }
+            },
+            "env": {
+              "EnableDiagnosticLogging": {
+                "value": "True"
+              }
+            }
+          },
+          "cv": {
```
Comment (naiteeks) [inline src/edge/deployment.customvision.template.json:88]: lets have a better name for the module:  maybe "customvision"

File: src/edge/deployment.customvision.template.json [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        "azureMediaServicesArmId": "/subscriptions/$SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP/providers/microsoft.media/mediaservices/$AMS_ACCOUNT",
+        "aadTenantId": "$AAD_TENANT_ID",
+        "aadServicePrincipalAppId": "$AAD_SERVICE_PRINCIPAL_ID",
+        "aadServicePrincipalSecret": "$AAD_SERVICE_PRINCIPAL_SECRET",
+        "aadEndpoint": "https://login.microsoftonline.com",
+        "aadResourceId": "https://management.core.windows.net/",
+        "armEndpoint": "https://management.azure.com/",
+        "diagnosticsEventsOutputName": "AmsDiagnostics",
+        "OperationalEventsOutputName": "AmsOperational"
+      }
```
Comment (naiteeks) [inline src/edge/deployment.customvision.template.json:140]: i think we should add the log co0llection properties as well

              "logLevel": "Information",
              "logCategories": "Application,Events",
              "allowUnsecuredEndpoints": true,
              "telemetryOptOut": false

File: src/edge/deployment.customvision.template.json [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      "properties.desired": {
+        "schemaVersion": "1.0",
+        "runtime": {
+          "type": "docker",
+          "settings": {
+            "minDockerVersion": "v1.25",
+            "loggingOptions": "",
+            "registryCredentials": {
+              "$CONTAINER_REGISTRY_USERNAME_myacr": {
+                "username": "$CONTAINER_REGISTRY_USERNAME_myacr",
```
Comment (naiteeks) [inline src/edge/deployment.customvision.template.json:14]: do we need these?

**Comment by gadamilan** [comment]
This is old and not relevant

**Pull request closed without merging**